app/components/search-result.js

```diff
@@ -27,6 +27,14 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    shortContributorList: Ember.computed('result', function() {
+        let filtered = this.get('result.contributors').filter(item => !!item.users.bibliographic);
+        let nameArr = filtered.slice(0, Math.min(10, filtered.length));
+        if (nameArr.length < filtered.length)
+            nameArr[nameArr.length - 1].originalLength = filtered.length;
```

Comment:
That said, this is a step in the right direction; much appreciated on the work so far. :)